documents/processes/review/EXAMPLE.md

```diff
@@ -0,0 +1,38 @@
+# AsyncAPI Review Document
+
+|Project name|Reviewers|Company|
+|------------|---------|-------|
+|Awesome project|Jack Johnson, John Jackson|ACME|
+
+## Our goals with this project are to:
+  1. Document a microservices architecture built with RabbitMQ.
+  2. Create an SDK that will take care of connecting, sending, and receiving messages.
+
+## We expect to:
+  1. Be able to define my queues and exchanges in AsyncAPI.
+  2. Have everything we need to make the SDK work.
+
+## We found the following challenges or difficulties:
+
+<!-- Please, repeat this block for every challenge/difficulty. -->
+
+### 1. The AMQP extension doesn't work for us.
+The AMQP extension doesn't allow me to define my queues and exchanges.
+
+#### We think it can be solved by...
+Improving the AMQP extension.
+
+#### Overall, we think this challenge/difficulty is...
+  * [ ] A "nice to have".
+  * [ ] A "must have" for the next minor version.
```

Comment:
Wondering if a rephrase of the must-haves might help, e.g. "A blocker for us to adopt v2 of the spec"? The reviewer may not have a good idea of which version it belongs in, but for us we need to know if this blocks them from adopting or if they can proceed with a workaround.

Reply:
Changed. Thanks for the feedback!